Tests/Rewriters/IgnoreArgumentsRewriterTests.cs

```diff
@@ -22,22 +22,19 @@ namespace RhinoMocksToMoqRewriter.Tests.Rewriters
   [TestFixture]
   public class IgnoreArgumentsRewriterTests
   {
-    [TestFixture]
-    public class MethodRewriterTests
-    {
-      private IgnoreArgumentsRewriter _rewriter;
+    private IgnoreArgumentsRewriter _rewriter;
```

Comment:
Whoa what happened here? Nice find, but this should be fixed in a separate commit.